zipkin/src/main/java/com/megaease/easeagent/zipkin/TracingProviderImpl.java

```diff
@@ -116,14 +115,12 @@ public Tracing tracing() {
     @Override
     public TracingSupplier tracingSupplier() {
         return supplier -> {
-            if (iTracing != null) {
-                return iTracing;
-            }
-            synchronized (TracingProviderImpl.class) {
-                if (iTracing != null) {
-                    return iTracing;
+            if (iTracing == null) {
+                synchronized (TracingProviderImpl.class) {
+                    if (iTracing == null) {
+                        iTracing = TracingImpl.build(supplier, tracing);
+                    }
```

Comment:
```java
if(xxx!=null){
    return xxx;
}
//init xxx
if(xxx!=null){
   return xxx;
}

//init xxx
if(xxx!=null){
   return xxx;
}
```
```java
if(xxx==null){
   //init xxx
   if(xxx==null){
      //init xxx  
       if(xxx==null){
          //init xxx
       }
   }
}
return xxx;
```
The first is more readable than the second, because the first is less nested.
At the same time they achieve the same purpose, so I don't understand why you modify this piece of code